### PR TITLE
Changing string formatting

### PR DESCRIPTION
**Pull request by alydersen**
Just some minor stuff, improving string formatting while I try to understand the code

### CODE REVIEW

**Comment by alydersen** [comment]
@magbak Friendly reminder to review this PR

File: opcua_tools/nodeset_generator.py [hunk 1]
```diff
@@ -83,6 +101,18 @@ def generate_references_xml(nodes, references):
...
+        references.loc[~references['target_same_ns'], 'Trg'])
+
+    # Incoming reference only when target is in same nodeset to be generated
+    references.loc[references['target_same_ns'], 'xml'] = "<Reference ReferenceType=\"{}\" IsForward=\"false\">{}</Reference>".format(
+        references.loc[references['target_same_ns'], 'ReferenceType'],
+        references.loc[references['target_same_ns'], 'Src'])
+
+
+    # Outgoing reference when target is not in same nodeset
+    print(~references['target_same_ns'])
```
Comment (magbak) [inline opcua_tools/nodeset_generator.py:115]: No need to print :)

File: opcua_tools/nodeset_generator.py [hunk 2]
```diff
@@ -83,6 +101,18 @@ def generate_references_xml(nodes, references):
...
 
     # Outgoing reference when target is not in same nodeset
+    references.loc[~references['target_same_ns'], 'xml'] = "<Reference ReferenceType=\"{}\">{}</Reference>".format(
+        references.loc[~references['target_same_ns'], 'ReferenceType'],
+        references.loc[~references['target_same_ns'], 'Trg'])
+
+    # Incoming reference only when target is in same nodeset to be generated
+    references.loc[references['target_same_ns'], 'xml'] = "<Reference ReferenceType=\"{}\" IsForward=\"false\">{}</Reference>".format(
+        references.loc[references['target_same_ns'], 'ReferenceType'],
+        references.loc[references['target_same_ns'], 'Src'])
```
Comment (magbak) [inline opcua_tools/nodeset_generator.py:111]: Same problem as above, are you sure this works?

**Review by magbak — CHANGES_REQUESTED** [review]

File: opcua_tools/nodeset_generator.py [hunk 3]
```diff
@@ -83,6 +101,18 @@ def generate_references_xml(nodes, references):
     references.loc[~references['target_same_ns'], 'NodeId'] = references.loc[~references['target_same_ns'], 'Src']
 
     # Outgoing reference when target is not in same nodeset
+    references.loc[~references['target_same_ns'], 'xml'] = "<Reference ReferenceType=\"{}\">{}</Reference>".format(
+        references.loc[~references['target_same_ns'], 'ReferenceType'],
+        references.loc[~references['target_same_ns'], 'Trg'])
```
[the remark this answers is not in the document]
Comment (alydersen) [reply opcua_tools/nodeset_generator.py]: Reverted - thanks

File: opcua_tools/nodeset_generator.py (hunk 2 above)
Comment (alydersen) [reply opcua_tools/nodeset_generator.py]: Reverted - thanks

**Review by magbak — APPROVED** [review]
Supert, dette ble mer oversiktlig

**Pull request merged**